src/methods/seek.js

```diff
@@ -29,6 +29,17 @@ const seek = ({ media, direction, seekSec, cacheRequired = false }) => {
   }
 }
 
+/**
+ * @param {HTMLVideoElement} media - Video element to be handled
+ * @param {number} numericKey - Number key pressed. Must be between 0 and 9
+ * @param {boolean} [cacheRequired=true] - `true` to force caching the new frame
+ * @returns {void}
+ */
+const decimalSeek = ({ media, numericKey, cacheRequired = true }) => {
+  media.currentTime = media.duration * (numericKey / 10)
```

Comment:
```suggestion
  const keyIndex = parseInt(numericKey)
  if (isNaN(keyIndex)) return

  media.currentTime = media.duration * (keyIndex / 10)
```

We should avoid implicit casting.